lib/main.js

```diff
@@ -122,12 +130,7 @@ module.exports = ({cwd}) => {
 					console.log(indentString(output, 1));
 				}
 			}
-
-			fs.unlinkSync(filePath('.dockerignore'));
-			if (originalDockerIgnore) {
-				fs.writeFileSync(filePath('.dockerignore', originalDockerIgnore));
-			}
-
+			dockerIgnoreCleanup();
 			return state;
 		});
```

Comment:
Why not just attaching a `.catch` handler at the end like this

```
.catch(err => {
    dockerCleanup();

    throw err;
});
```

This way, we catch all errors and always clean up if something goes wrong. Not only te `isDockerUp` error.